File: backend-spring/src/main/java/io/github/agussyahrilmubarok/backend/application/dto/user/UserResponse.java
package io.github.agussyahrilmubarok.backend.application.dto.user;

import com.fasterxml.jackson.annotation.JsonProperty;
import java.time.OffsetDateTime;
import java.util.UUID;

public record UserResponse(
        @JsonProperty("id") UUID id,

        @JsonProperty("name") String name,

        @JsonProperty("email") String email,

        @JsonProperty("created_at") OffsetDateTime createdAt,

        @JsonProperty("updated_at") OffsetDateTime updatedAt) {}

File: examples/backend-crud/backend-dotnet/Backend/Application/DTO/User/UserResponse.cs
using System.Text.Json.Serialization;

namespace Backend.Application.DTO.User;

public record UserResponse(
    [property: JsonPropertyName("id")]
    Guid Id,

    [property: JsonPropertyName("name")]
    string Name,

    [property: JsonPropertyName("email")]
    string Email,

    [property: JsonPropertyName("created_at")]
    DateTime CreatedAt,

    [property: JsonPropertyName("updated_at")]
    DateTime UpdatedAt
);